internal/services/bot/bot_channel_direct_line_speech_resource.go

```diff
@@ -64,6 +65,12 @@ func resourceBotChannelDirectLineSpeech() *pluginsdk.Resource {
 
 			"cognitive_service_location": commonschema.LocationWithoutForceNew(),
 
+			"cognitive_service_id": {
```

Comment:
Since the resource is `azurerm_cognitive_account` I think we should rename this to `cognitive_account_id` instead of `cognitive_service_id`.
```suggestion
			"cognitive_account_id": {
```

Reply:
Updated